litgpt/deploy/serve.py

```diff
@@ -0,0 +1,133 @@
+from pathlib import Path
+from typing import Dict, Any, Optional
+from litgpt.utils import check_valid_checkpoint_dir
+
+import lightning as L
+import torch
+from litserve import LitAPI, LitServer
+
+from litgpt.model import GPT
+from litgpt.config import Config
+from litgpt.tokenizer import Tokenizer
+from litgpt.generate.base import generate
+from litgpt.prompts import load_prompt_style, has_prompt_style, PromptStyle
+from litgpt.utils import load_checkpoint, CLI
+
+
+class SimpleLitAPI(LitAPI):
+
+    def __init__(self,
+                 checkpoint_dir: Path,
+                 precision: Optional[str] = None,
+                 temperature: float = 0.8,
+                 top_k: int = 200,
```

Comment:
This is a pretty high default, I'd choose a smaller one (openAI has 50)

Reply:
I agree. I think 20 is actually common. I took the 200 from the `litgpt generate` defaults here: https://github.com/Lightning-AI/litgpt/blob/410a7126f82ea550d4a43dab89367547b073b5a3/litgpt/generate/base.py#L99

Reply:
Let's standardize around 20 or 50, but let's compare 20 vs 50 first and see if there are material differences

Reply:
I ran couple of experiments and the difference between 50 and 200 is practically not really noticeable due to the small temperature. However, let's standardize on 50 if that's what other popular LLMs like GPT4 use.
Ollama uses 100 with temperature 2.5, Axolotl uses 40 with temperature 0.9 btw. 

In any case, the important thing is that the defaults work reasonably well and, users can still easily override that.